apps/krystal-bull/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  web:
+    image: bitcoinscala/oracle-server-ui:0.0.1-f4c297c5-SNAPSHOT
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      - ORACLE_SERVER_API_URL=http://10.21.21.61:9998/
```

Comment:
```suggestion
      - ORACLE_SERVER_API_URL: http://10.21.21.61:9998/
```

Reply:
Needs to lose the leading dash as well.

Reply:
Actually, both are valid and used inconsitently in Umbrel.

Docs: https://docs.docker.com/compose/compose-file/compose-file-v3/#environment

Examples of this format: https://github.com/getumbrel/umbrel/blob/c55c60bf22e430e53ddeb3733bd8bad5e85a8976/apps/pi-hole/docker-compose.yml#L18 https://github.com/getumbrel/umbrel/blob/c55c60bf22e430e53ddeb3733bd8bad5e85a8976/apps/nextcloud/docker-compose.yml#L44
Example of the other format: https://github.com/getumbrel/umbrel/blob/c55c60bf22e430e53ddeb3733bd8bad5e85a8976/apps/bluewallet/docker-compose.yml#L29